### PR TITLE
Fix/LP-187 DropDownTextField length and color

### PR DESCRIPTION
**Pull request by JoseGR98**
LP-187
https://devaptivist.atlassian.net/browse/LP-187
DropDownTextField composable was updated. Background color updated to white. Length was updated. Code refactored. Some improvements on arrow icon. Some elements changed to be able to make the feature. Not warnings or errors showed.

Two TextField using the DropDownTextField in RepairView
Screenshot 2023-05-02 173249

DropDownTextField - PetInUnit field
Screenshot 2023-05-02 173036

DropDownTextField - Category field
Screenshot 2023-05-02 173052

### CODE REVIEW

**Comment by JoseGR98** [comment]
Automatic check is in progress. After that, waiting for code review.
@carlosph816 @melissaquezada94

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt [hunk 1]
```diff
@@ -285,24 +284,23 @@ fun DropdownTextField(
...
+        },
         modifier = modifier
     ) {
         OutlinedTextField(
             modifier = Modifier
                 .fillMaxWidth()
                 .menuAnchor(),
             value = selectedOptionText,
-            onValueChange = {},
+            onValueChange = { },
```
Comment (carlosph816) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt]: Let's put and "TODO" to know in the future what we are going to do

Comment (melissaquezada94) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt]: Please add a text regarding what is going to do inside the `TODO()`

Comment (JoseGR98) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt]: Sure, added it!

**Comment by JoseGR98** [comment]
TODO added based on code reviewers comments. Pending for new checks

**Review by melissaquezada94 — APPROVED** [review]
Nicely job!

**Pull request merged**